doc/release-notes.md

```diff
@@ -40,301 +40,69 @@ unsupported systems.
-
-- The binary verification script has been updated. In previous releases it
-  would verify that the binaries had been signed with a single "release key".
-  In this release and moving forward it will verify that the binaries are
-  signed by a _threshold of trusted keys_. For more details and
-  examples, see:
-  https://github.com/bitcoin/bitcoin/blob/master/contrib/verify-binaries/README.md
-  (#27358)
-
-Low-level changes
-=================
-
-RPC
----
-
-- The JSON-RPC server now rejects requests where a parameter is specified multiple
-  times with the same name, instead of silently overwriting earlier parameter values
-  with later ones. (#26628)
-- RPC `listsinceblock` now accepts an optional `label` argument
-  to fetch incoming transactions having the specified label. (#25934)
-- Previously `setban`, `addpeeraddress`, `walletcreatefundedpsbt`, methods
-  allowed non-boolean and non-null values to be passed as boolean parameters.
-  Any string, number, array, or object value that was passed would be treated
-  as false. After this change, passing any value except `true`, `false`, or
-  `null` now triggers a JSON value is not of expected type error. (#26213)
+- #27777 ci: Prune dangling images on RESTART_CI_DOCKER_BEFORE_RUN
+- #27834 ci: Nuke Android APK task, Use credits for tsan
+- #27844 ci: Use podman stop over podman kill
+- #27886 ci: Switch to amd64 container in "ARM" task
 
 Credits
 =======
 
 Thanks to everyone who directly contributed to this release:
 
-- 0xb10c
-- 721217.xyz
-- @RandyMcMillan
-- amadeuszpawlik
-- Amiti Uttarwar
+- Abubakar Sadiq Ismail
 - Andrew Chow
-- Andrew Toth
-- Anthony Towns
-- Antoine Poinsot
-- Aurèle Oulès
-- Ben Woosley
-- Bitcoin Hodler
-- brunoerg
-- Bushstar
-- Carl Dong
-- Chris Geihsler
-- Cory Fields
-- David Gumberg
-- dergoegge
-- Dhruv Mehta
-- Dimitris Tsapakidis
-- dougEfish
-- Douglas Chimento
-- ekzyis
-- Elichai Turkel
-- Ethan Heilman
-- Fabian Jahr
-- FractalEncrypt
-- furszy
-- Gleb Naumenko
-- glozow
-- Greg Sanders
+- Bruno Garcia
+- Gregory Sanders
 - Hennadii Stepanov
-- hernanmarino
-- ishaanam
-- ismaelsadeeq
-- James O'Beirne
-- jdjkelly@gmail.com
-- Jeff Ruane
-- Jeffrey Czyz
-- Jeremy Rubin
-- Jesse Barton
-- João Barbosa
-- JoaoAJMatos
-- John Moffett
-- Jon Atack
-- Jonas Schnelli
-- jonatack
-- Joshua Kelly
-- josibake
-- Juan Pablo Civile
-- kdmukai
-- klementtan
-- Kolby ML
-- kouloumos
-- Kristaps Kaupe
-- laanwj
-- Larry Ruane
-- Leonardo Araujo
-- Leonardo Lazzaro
-- Luke Dashjr
 - MacroFake
-- MarcoFalke
-- Martin Leitner-Ankerl
-- Martin Zumsande
-- Matt Whitlock
-- Matthew Zipkin
+- Matias Furszyfer
 - Michael Ford
-- Miles Liu
-- mruddy
-- Murray Nesbitt
-- muxator
-- omahs
-- pablomartin4btc
-- Pasta
 - Pieter Wuille
-- Pttn
-- Randall Naar
-- Riahiamirreza
-- roconnor-blockstream
-- Russell O'Connor
-- Ryan Ofsky
-- S3RK
-- Sebastian Falbesoner
-- Seibart Nedor
-- sinetek
-- Sjors Provoost
-- Skuli Dulfari
-- SomberNight
-- Stacie Waleyko
 - stickies-v
```

Comment:
nit: Mention by name, as all others contributors?

Reply:
Copied from the previous release notes.

Reply:
There are also other usernames in here.

Reply:
> There are also other usernames in here.

![image](https://github.com/bitcoin/bitcoin/assets/32963518/92edbf07-8f36-4f13-b0df-019c9c296698)

Reply:
> MacroFake